firmware/boron-ins-fsm/src/im21door.cpp

```diff
@@ -153,30 +153,23 @@ void threadBLEScanner(void *param) {
   BLE.setScanTimeout(5);
 
   while(true){
-
-    int count = BLE.scan(scanResults, SCAN_RESULT_MAX);
```

Comment:
Can `scanResults` and `SCAN_RESULT_MAX` now be removed? This is the only place you're using them right?